distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -201,6 +202,7 @@ grant codeBase "file:/platform-osgi-configadmin" {
 
 grant codeBase "file:/org.apache.karaf.deployer.blueprint/org.apache.karaf.deployer.kar" {
     permission java.io.FilePermission "${ddf.home.perm}etc${/}definitions${/}-", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}-", "read";
```

Comment:
This line change will go away once https://github.com/codice/ddf/pull/4827 is merged.